### PR TITLE
ECR + helm image check

### PR DESCRIPTION
**Pull request by martinkersner**
# Description

Check on ECR + helm image check. It should help with daily check and avoid accidentally deleting currently used images from repository. It will need checks for VRF, RR, and Aggregator image tags, but it is good first step.

```
❯❯❯ ./ecr-check.sh
orakl-core: count FAIL (7 tags)
(cypress) orakl-core: vrf FAIL (v0.3.1.20231103.0653.19ca105)
(cypress) orakl-core: request-response FAIL (v0.3.1.20231016.1000.749dac8)
(cypress) orakl-core: aggregator OK (v0.5.1.20240122.0131.44014e7)
(baobab) orakl-core: vrf OK (v0.5.1.20240111.1440.dacb2ef)
(baobab) orakl-core: vrf OK (v0.5.1.20240111.1440.dacb2ef)
(baobab) orakl-core: vrf OK (v0.5.1.20240111.1440.dacb2ef)
(baobab) orakl-core: request-response OK (v0.5.1.20240111.1440.dacb2ef)
(baobab) orakl-core: request-response OK (v0.5.1.20240122.0240.1672788)
(baobab) orakl-core: request-response OK (v0.5.1.20240111.1440.dacb2ef)
(baobab) orakl-core: aggregator OK (v0.5.1.20231221.0205.5ff1278)
(baobab) orakl-core: aggregator OK (v0.5.1.20240122.0240.1672788)
(baobab) orakl-core: aggregator OK (v0.5.1.20240122.0131.44014e7)
orakl-api: count OK
(cypress) orakl-api: api OK (v0.1.0.20231214.0553.3dd6e26)
(baobab) orakl-api: api OK (v0.1.0.20231214.0451.cb4bd3a)
orakl-cli: count OK
(cypress) orakl-cli: cli FAIL (v0.6.0.20230920.0750.99c4cdd)
(baobab) orakl-cli: cli OK (v0.6.0.20231229.0312.9433d53)
orakl-fetcher: count FAIL (6 tags)
(cypress) orakl-fetcher: fetcher OK (v0.0.1.20231221.0248.5ff1278)
(baobab) orakl-fetcher: fetcher OK (v0.0.1.20240122.0240.1672788)
orakl-delegator: count OK
(cypress) orakl-delegator: delegator FAIL (v0.0.1.20230707.0137.513e9f9)
(baobab) orakl-delegator: delegator OK (v0.0.1.20231211.0735.867d885)
```

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Deployment

- [ ] Should publish npm package
- [ ] Should publish Docker image

### CODE REVIEW

File: monitor/scripts/ecr-check.sh [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            continue
+        fi
+
+        check_tag ${tag}
+    done
+}
+
+check_tag() {
+    tag="${1}"
+
```
Comment (nick-bisonai) [inline monitor/scripts/ecr-check.sh:58]: can we have additional parameter such as `tag_name` to print out if it is l, w, or r?

so maybe

```
check_tag() {
tag_name="$2"
....
if [ "$found_tag" = false ]; then
        echo "(${chain}) ${repository_name}: ${service_name} FAIL ($tag):($tag_name)"
    else
        echo "(${chain}) ${repository_name}: ${service_name} OK ($tag):($tag_name)"
    fi
...
}
```

Comment (martinkersner) [reply monitor/scripts/ecr-check.sh:58]: Good idea, let me add it!

Comment (martinkersner) [reply monitor/scripts/ecr-check.sh:64]: https://github.com/Bisonai/orakl/pull/1071/commits/5c685c97806eb7581cc2e6774cb3c0e28729d94f

**Review by nick-bisonai — APPROVED** [review]
LGTM!

**Review by jay-bisonai — APPROVED** [review]
LGTM

**Pull request merged**